### PR TITLE
Changelog 23/11

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Changelog entries:

- 2FA
- Stand-alone Devices in Pipeline
- Project Nodes for Devices
- "Open Editor" enabled by default when in Developer Mode

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/changelog/2023/11/devices-in-pipelines.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Devices in DevOps Pipelines
+date: 2023-11-23 12:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+Building on our [previous release](https://flowfuse.com/blog/2023/08/flowfuse-1-11-release/#usability-improvements-to-device-management-%232294) that enabled the management of devices independently of instances. Now, the integration of individual devices into DevOps Pipelines is possible. This update marks a significant step towards more flexible and efficient device management within FlowFuse. In our ongoing efforts, we are also developing features to group devices, aiming to streamline and improve the overall device management experience.
```
Comment (MarianRaphael) [inline src/changelog/2023/11/devices-in-pipelines.md:8]: I will add an image

File: src/changelog/2023/11/2fa.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Two-Factor Authentication
+date: 2023-11-23 13:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+In our ongoing commitment to strengthen security, we're excited to announce the addition of two-factor authentication (2FA) for FlowFuse. This enhancement ensures an extra layer of security for your account. Setting up 2FA is straightforward, and the system supports all commonly used TOTP (Time-Based One-Time Password) tools, such as Google Authenticator. To enable 2FA, simply navigate to your User Settings and follow the easy setup process.
+
```
Comment (ZJvandeWeg) [inline src/changelog/2023/11/2fa.md:10]: Let's add a link to the docs for each changelog entry

Comment (ZJvandeWeg) [reply src/changelog/2023/11/2fa.md:10]: And for each item let's make clear which tiers it's available in

File: src/changelog/2023/11/default-editor.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Device Editor enabled by default
+date: 2023-11-23 11:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+With the latest update, devices operating in `Developer Mode` will now have the `Open Editor` feature enabled by default. This enhancement is designed to improve the User Experience.
```
Comment (ZJvandeWeg) [inline src/changelog/2023/11/default-editor.md:9]: Isn't it the other way around? For each device there's a editor button which will enable developer mode implicitly

File: src/changelog/2023/11/project-nodes-devices.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: Project Nodes for Devices
+date: 2023-11-23 11:02:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+ From this update onwards, Project Nodes will now be available for devices assigned to an application. Previously, it was necessary for devices to be bound to an instance to utilize this functionality.
```
Comment (ZJvandeWeg) [inline src/changelog/2023/11/project-nodes-devices.md:9]: Explain the difference between an instance and an application please

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/changelog/2023/11/project-nodes-devices.md (hunk 4 above)
Comment (MarianRaphael) [reply src/changelog/2023/11/project-nodes-devices.md:9]: [difference between Instances, Devices, and Applications](https://github.com/FlowFuse/website/pull/1338/commits/01826c568eb45000c9337a430fce6d349cb5e7b9)

File: src/changelog/2023/11/default-editor.md (hunk 3 above)
Comment (MarianRaphael) [reply src/changelog/2023/11/default-editor.md:9]: No

Comment (MarianRaphael) [reply src/changelog/2023/11/default-editor.md:9]: Screenshot 2023-11-23 at 11 43 52
Screenshot 2023-11-23 at 11 44 03

File: src/changelog/2023/11/2fa.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+![Config 2FA](./images/2fa.png)
+
+For more Details see our [Documentation](https://flowfuse.com/docs/user/user-settings/)
```
Comment (ZJvandeWeg) [inline src/changelog/2023/11/2fa.md:13]:
```suggestion
For more Details see our [Documentation](/docs/user/user-settings/)
```

File: src/changelog/2023/11/project-nodes-devices.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+ - [Click here](https://flowfuse.com/docs/user/projectnodes/#flowfuse-project-nodes) to leran more about the Project Nodes.
```
Comment (ZJvandeWeg) [inline src/changelog/2023/11/project-nodes-devices.md:11]:
```suggestion
 - [Learn more about the Project nodes](/docs/user/projectnodes/#flowfuse-project-nodes)
```

File: src/changelog/2023/11/project-nodes-devices.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+title: Project Nodes for Devices
+date: 2023-11-23 11:02:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+ From this update onwards, Project Nodes will now be available for devices assigned to an application. Previously, it was necessary for devices to be bound to an instance to utilize this functionality.
```
Comment (ZJvandeWeg) [inline src/changelog/2023/11/project-nodes-devices.md:9]: It's a changelog, there's no update anymore, it's effective today.

Comment (ZJvandeWeg) [reply src/changelog/2023/11/project-nodes-devices.md:9]:
```suggestion
From today onwards, Project Nodes will now be available for devices assigned to an application. Previously, it was necessary for devices to be bound to an instance to utilize this functionality.
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**